src/views/dialogs/DepositDialog2/DepositStatus.tsx

```diff
@@ -18,11 +27,26 @@ type DepositStatusProps = {
 // TODO(deposit2.0): localization for this whole component
 export const DepositStatus = ({ txHash, chainId, onClose }: DepositStatusProps) => {
   const deposit = useParameterizedSelector(selectDeposit, txHash, chainId);
+  const { freeCollateral } = orEmptyObj(
+    useAppSelector(BonsaiCore.account.parentSubaccountSummary.data)
+  );
+  const initialFreeCollateral = useRef(freeCollateral);
```

Comment:
would there ever be a race condition if the user were able to open the dialog before freeCollateral is populated or is pending?

Reply:
yeah, i considered in this case it's fine to just have the success state happen whenever any sort of free collateral value loads